### ISSUE TITLE
Renaming a data table does not work

### ISSUE THREAD

**Issue by Franco98404** [FlutterFlow/flutterflow-issues#5015]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Renaming a data table does not work

### Expected Behavior

It should be possible to rename the Data Table widget like other widgets

### Steps to Reproduce

1. Add the DataTable widget
2. Try to change the name and nothing happens

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4ClPLq25dIsctZ1rrtbcBvmiUsJjsjRYJMmsFTTRcbJ5P8C5kAYczBM0poTdO5fHxIeVqYmmMJ+eDThIbXGMABAE+qRqZ73JVqTzziQj6hSqaYPJDge3ZfHcVYG1Sw4Kugnh1WA+FhV0Znw3GIJfPKVgDfQrruDWwsRvj+Nr7UjEK/GCLNAids6G0XAx7eTI5ODEz25eKMrop5yfj/0w==

### Visual documentation

Try to rename and nothing happens

![image](https://github.com/user-attachments/assets/b3170780-6e51-4136-8ef0-c8748e9355d3)

### Environment

```markdown
- FlutterFlow version: FlutterFlow v5.0.13+ released November 27, 2024 Flutter version is 3.24.2
- Platform: Web
- Browser name and version: Chrome
- Operating system and version affected: W11
```

**Comment by Alezanello** [comment]
You're absolutely right. I'll forward this to the engineering team so they can address the issue.

**Comment by icestormpure** [comment]
Still doesn't work

**Comment by icestormpure** [comment]
> You're absolutely right. I'll forward this to the engineering team so they can address the issue.

This still doesn't work after recent update
